Library/Homebrew/rubocops/components_order.rb

```diff
@@ -112,7 +112,7 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
         end
 
         def check_on_system_block_content(component_precedence_list, on_system_block)
-          if on_system_block.body.block_type? && !on_system_methods.include?(on_system_block.body.method_name)
+          if on_system_block.body.block_type? && on_system_methods.exclude?(on_system_block.body.method_name)
```

Comment:
Not sure if this is also a false positive, as in the other violation in this directory below. We might want to disable it anyway, as we have comments elsewhere to avoid using ActiveSupport in rubocops (though we make extensive use of `present?`)

Reply:
@dduugg seems to be: https://github.com/Homebrew/brew/actions/runs/4305286476/jobs/7507553493#step:6:12